src/Shell/QueueShell.php

```diff
@@ -163,7 +163,7 @@ public function runworker() {
 				$taskname = 'Queue' . $queuedTask['job_type'];
 
 				try {
-					$data = json_decode($queuedTask['data'], true);
+					$data = unserialize($queuedTask['data']);
```

Comment:
This broke master, the change here was unrelated and invalid :/

Reply:
I was kind of under the impression we only changed the email task here for it, not changing all of them.. not sure if thats good or not..